ModelBundle/Migrations/MongoDB/Version20160511093334.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace OpenOrchestra\ModelBundle\Migrations\MongoDB;
+
+use AntiMattr\MongoDB\Migrations\AbstractMigration;
+use Doctrine\MongoDB\Database;
+
+/**
+ * Class Version20160511093334
+ */
+class Version20160511093334 extends AbstractMigration
+{
+    /**
+     * @return string
+     */
+    public function getDescription()
+    {
+        return "update attributes of block configurable content";
+    }
+
+    /**
+     * @param Database $db
+     */
+    public function up(Database $db)
+    {
+        $db->execute('
+            db.node.find().forEach(function(item) {
+                for (key in item.blocks) {
+                    block = item.blocks[key];
+                    contentSearch = {}
+                    if (block.component == "configurable_content" &&
+                        typeof block.attributes["contentTypeId"] !== "undefined" &&
+                        typeof block.attributes["contentId"] !== "undefined" &&
+                        typeof block.attributes["contentSearch"] == "undefined"
+                    )
+                    {
```

Comment:
this one is on 2 lines, the second line 64 is on one line, that should be coherent